Gemfile

```diff
@@ -3,9 +3,9 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 gemspec
 
 if facterversion = ENV['FACTER_GEM_VERSION']
-  gem 'facter', facterversion, :require => false
+  gem 'facter', facterversion, require: false
 else
-  gem 'facter', :require => false
+  gem 'facter', require: false
 end
```

Comment:
We should just solve the TODO and make it:
```ruby
gem 'facter', ENV['FACTER_GEM_VERSION'], require: false
```

Though perhaps that fails another cop and you should use

```ruby
gem 'facter', ENV.fetch('FACTER_GEM_VERSION', nil), require: false
```

Reply:
done